cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -275,32 +283,45 @@ def test_validate_memory_usage(resource_usage, metering_report):
             usage_from_report = groups["Memory Used"]
             if 'GB' in usage_from_report:
                 estimated_memory_usage = estimated_memory_usage * math.pow(2, -10)
-            usage = re.sub(r'[MB, GB,]', r'', usage_from_report)
+            usage = re.sub(r'(MB|GB)', r'', usage_from_report)
             assert estimated_memory_usage - DEVIATION <= float(usage) \
                 <= estimated_memory_usage + DEVIATION, 'Estimated cost and report cost do not match'
             break
 
 
 def test_validate_network_usage(resource_usage, metering_report):
-    """Test to validate network usage cost."""
+    """Test to validate network usage."""
     for groups in metering_report:
         if groups["Network I/O Used"]:
             estimated_network_usage = resource_usage['network_io']
             usage_from_report = groups["Network I/O Used"]
-            usage = re.sub(r'[KBps,]', r'', usage_from_report)
+            usage = re.sub(r'(KBps)', r'', usage_from_report)
```

Comment:
OPTIONAL: ``usage_from_report.replace('KBps', '')`` is enough here

Reply:
I have changed it to this , since the comma also needs to be stripped.
usage = re.sub(r'(KBps|,)', r'', usage_from_report)